[rel-5_0] Ported back translatable messages from master to rel-5_0

urbalazs opened the pull request:
Hi @mgruner 
I reviewed the Kernel/Modules/*.pm files in rel-5_0, and I ported back the translatable strings from master. I also ported back this commit: https://github.com/OTRS/otrs/commit/99f85458b8d9011b808c25b0ebebd655ea820ba2

mgruner commented on Kernel/Output/HTML/Templates/Standard/CustomerLogin.tt:
```diff
@@ -115,8 +115,8 @@
                     <div>
-                        <label for="Title">[% Translate("Title") | html %]</label>
-                        <input title="[% Translate("How we should address you") | html %]" type="text" name="Title"  id="Title" maxlength="50" value="[% Data.UserTitle | html %]"/>
+                        <label for="Title">[% Translate("Title or salutation") | html %]</label>
```
These seem to be unrelated changes that should go into another PR or bugfix.

mgruner commented on var/httpd/htdocs/skins/Customer/default/css/Core.Login.css:
```diff
@@ -178,10 +178,7 @@
-    width: 55px;
-}
-
+#Signup #Title,
```
This is also about a different matter.

mgruner commented:
Hello @urbalazs. This PR contains unrelated changes, please check. Besides that, I would like to ask you to provide the changes for rel-5_0 in smaller chunks of a few files each for better testing/review possibilities. I already burnt my fingers with cherrypicking such changes from master...

I'll close this huge PR for now. Looking forward to more PRs from you, we'll make sure to get your changes into rel-5_0!

urbalazs commented:
Hi @mgruner,
Thank you for the suggestion, next time I will make smaller PRs. I removed from this one the backporting of 'Title or salutation' issue from these files: AdminUser.tt, CustomerLogin.tt, Core.Login.css.
May I reopen this huge PR (last time), or should I split it into smaller ones? This PR contains only modifications, which are currently in master.

mgruner commented:
I would be grateful if you could create a handful of smaller PRs from this, please.
